src/components/LoginScreen.js

```diff
@@ -16,11 +17,13 @@ export class LoginScreen extends Component {
         super();
         this.springValue = new Animated.Value(0.3);
         this.rotateInterpolate = new Animated.Value(0);
+        this.state = {
```

Comment:
There is no need to store animated value instance in the state because it's not displayed directly on the layout and will be passed to the Animated components, who listens for the updates of animated values by themselves. So, because you're not invoke super of parent class with custom parameters, this code could be rewritten from:
```
export class LoginScreen extends Component {    
//...
constructor(){
        super();
        this.springValue = new Animated.Value(0.3);
        this.rotateInterpolate = new Animated.Value(0);
        this.state = {
          progress: new Animated.Value(0),
        };
    }
//...
}   
```
to
```
export class LoginScreen extends Component {    
//...

    springValue = new Animated.Value(0.3);
    rotateInterpolate = new Animated.Value(0);
    progress = new Animated.Value(0);

//...
}   
```